src/mkdocstrings/handlers/python.py

```diff
@@ -235,18 +237,24 @@ class PythonHandler(BaseHandler):
     """The Python handler class, nothing specific here."""
 
 
-def get_handler(theme: str, custom_templates: Optional[str] = None) -> PythonHandler:
+def get_handler(
+    theme: str, custom_templates: Optional[str] = None, setup_cmd_list: Optional[List[str]] = None
```

Comment:
Additionally accept a `**config` to be compatible with the "parent" `get_handler` (see previous comments).
Or only accept `**config` and pop `setup_cmd_list` from it :slightly_smiling_face: